src/solvers/dgsem_p4est/containers_3d.jl

```diff
@@ -10,7 +10,7 @@ function init_elements!(elements, mesh::P4estMesh{3}, basis::LobattoLegendreBasi
   @unpack node_coordinates, jacobian_matrix,
           contravariant_vectors, inverse_jacobian = elements
 
-  calc_node_coordinates!(node_coordinates, mesh, basis.nodes)
+  calc_node_coordinates!(node_coordinates, mesh, basis)
```

Comment:
I think we call some `calc_node_coordinates!` from Trixi.jl in Trixi2Vtk.jl, e.g., https://github.com/trixi-framework/Trixi2Vtk.jl/blob/16057193f5ea4b3b1b9e5cbe8d4962e2e16e1c31/src/vtktools.jl#L186. Does this interfere with this change?
CC @efaulhaber

Reply:
Aha, gotcha. I noticed that the routine below the constructor with the `@ assert` about the two `polydeg`s was never entered. That is, it was possible to run with `mesh_polydeg > solver_polydeg`.